plugins/modules/zos_script.py

```diff
@@ -112,6 +100,12 @@
   - When executing local scripts, temporary storage will be used
     on the remote z/OS system. The size of the temporary storage will
     correspond to the size of the file being copied.
+  - The location in the z/OS system where local scripts will be copied to can be
+    configured through Ansible's C(remote_tmp) option. Refer to
+    L(Ansible's documentation,https://docs.ansible.com/ansible/latest/collections/ansible/builtin/sh_shell.html#parameter-remote_tmp)
```

Comment:
curious why the link here is to the shell module (which seems to be an example use case of the remote_tmp environment var)


Maybe the link here should be one of the following:
- https://docs.ansible.com/ansible/latest/collections/environment_variables.html#envvar-ANSIBLE_REMOTE_TEMP
- https://docs.ansible.com/ansible/latest/collections/environment_variables.html#envvar-ANSIBLE_REMOTE_TMP

Not sure what the difference between tmp and temp is...

BUT the link you provided may also be the correct one. If the unix shell plugin is the only one in the list that's applicable, it may be nicer to point users to a more detailed description of the remote_tmp var (current link) over the links i've provided above which point to the generic env vars in the list of all ansible env vars
![image](https://github.com/ansible-collections/ibm_zos_core/assets/10567394/d9323e5a-65e2-4390-8c1f-4956e16ffe9d)

Reply:
If I recall `remote_tmp` was at one time the replacement for `remote_temp` which never seemed to have happened. Back in ansible 2.5 they had this in the release notes:
```
The ANSIBLE_REMOTE_TMP environment variable has been added to supplement (and override) ANSIBLE_REMOTE_TEMP. This matches with the spelling of the config value. ANSIBLE_REMOTE_TEMP will be deprecated in the future
```

I believe most the plugins eventually discuss the env var `remote_tmp` or `remote_temp` and point to the link you suggested above (https://docs.ansible.com/ansible/latest/collections/environment_variables.html#envvar-ANSIBLE_REMOTE_TMP) and (https://docs.ansible.com/ansible/latest/collections/environment_variables.html#envvar-ANSIBLE_REMOTE_TEMP).

Reply:
I agree that linking to the environment variable directly is probably more correct, but the shell module has a more in-depth explanation of what it does, while also linking to the environment vars docs.